src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowWorkerBuilding.java

```diff
@@ -24,6 +26,18 @@
      */
     private static final String BUTTON_HIRE = "hire";
 
+    
+    /**
+     * Id of the scroll view
+     */
+    private static final String LIST_WORKERS = "workers";
+
+    /**
+     * ScrollList with the workers.
+     */
+    private ScrollingList recipeList;
```

Comment:
### Errors:
- Type of field 'recipeList' is concrete class ScrollingList 
- Instance field recipeList may not be initialized during object construction 
- Field can be converted to a local variable